_cite/cite.py

```diff
@@ -63,37 +64,43 @@
 
             # run plugin on data entry to expand into multiple sources
             try:
-                entry = import_module(f"plugins.{plugin.stem}").main(entry)
+                expanded = import_module(f"plugins.{plugin.stem}").main(entry)
                 # check that plugin returned correct format
-                if not list_of_dicts(entry):
+                if not list_of_dicts(expanded):
                     raise Exception("Plugin didn't return list of dicts")
             except Exception as e:
+                print(traceback.format_exc())
                 log(e, 3, "ERROR")
                 error = True
                 continue
 
             # loop through sources
-            for source in entry:
+            for index, source in enumerate(expanded):
```

Comment:
Ah yes, I was experimenting having `index` as a "debug field" on the citation like `["plugin"]` and `["file"]`. I thought it might help people to show them which "entry number" in a yaml file an error occurred at. But then I thought that that's kinda useless compared to showing the `id:` field (what I currently do), which can easily be ctrl+f'd. I also thought about [including actual yaml line number, but couldn't figure it out until I got help recently](https://stackoverflow.com/questions/76258698/pyyaml-customize-loader-to-include-depth-level), so I might include that in a future PR. I'll remove this index and enumerate.